source/declarators.tex

```diff
@@ -55,51 +55,45 @@
 \pnum
 Each
 \grammarterm{init-declarator}
-in a declaration is analyzed separately as if it was in a declaration by
-itself.\footnote{A declaration with several declarators is usually equivalent
-to the corresponding sequence of declarations each with a single declarator.
-That is
-
-\tcode{T  D1, D2, ... Dn;}
-
-\noindent is usually equivalent to
-
-\tcode{T  D1; T D2; ... T Dn;}
-
-\noindent where
-\tcode{T}
-is a
-\grammarterm{decl-specifier-seq}
-and each
-\tcode{Di}
-is an
-\grammarterm{init-declarator}.
+in a declaration is analyzed separately as if it was in a declaration by itself.
+\begin{note}
+A declaration with several declarators is usually equivalent to the corresponding
+sequence of declarations each with a single declarator. That is
+\begin{codeblock}
+T D1, D2, ... Dn;
+\end{codeblock}
+is usually equivalent to
+\begin{codeblock}
+T D1; T D2; ... T Dn;
+\end{codeblock}
+where \tcode{T} is a \grammarterm{decl-specifier-seq}
+and each \tcode{Di} is an \grammarterm{init-declarator}.
 An exception occurs when a name introduced by one of the
```

Comment:
If we're promoting this to a note, the bar for clarity and correctness is a little higher, and we shouldn't be saying "an exception occurs when" unless we mean that an exception is thrown. How about "One exception is when" here and "Another exception is when" below?